docs/Tutorials/Visualization.md

```diff
@@ -2,26 +2,81 @@
-script `Support/bin/Render1dData.py`. Run
-`python /path/to/Render1dData.py --help` to view the help text for the script.
-Let us consider a Burgers evolution where we want to visualize the solution,
-`U`, then in the run directory you can use
+SpECTRE source files for evolution executables are located in
+`src/Evolution/Executables`. Executables can be compiled by running the command
+`make EXECUTABLE_NAME` where `EXECUTABLE_NAME` is the name of the executable
+as defined in the `CMakeLists.txt` file located in the same directory as the
+source file. For example, to compile the executable that evolves a scalar wave
+using a three-dimensional domain, one runs the command:
+`make EvolveScalarWave3D`, which then results in an executable of the same name
+in the `bin` directory of the user's build directory.
 
-### 2D & 3D %Data Volume %Data In ParaView
+### Running an Evolution Executable
 
-Once you have generated volume data you must run the
-executable `ApplyObservers` in the run directory to generate an XDMF
-(`.xmf` extension) file. Open the `VisVolumeData.xmf` file in ParaView
-and selet the `Xdmf Reader`, *not* the version 3 readers. On the left hand side
-of the main ParaView window is a section named `Properties`, here you must
-click the highlighted `Apply` button. ParaView will now render your volume data.
+Each SpECTRE executable reads a user-provided YAML file that specifies the
+runtime settings of the evolution. This is where options such as the Domain,
+AnalyticSolution, TimeStepper, etc are supplied. Example input files are kept
+in `tests/InputFiles`, and are written to provide a functional base input file
+which the user can then modify as desired. Copy the executable and YAML file
+to a directory of your choice. The YAML file is then passed as an argument to
+the executable using the flag `--input-file`. For example, for a scalar wave
+evolution, run the command:
+`./EvolveScalarWave3D --input-file Input3DPeriodic.yaml`.
+By default, the example input files do not produce any output. This can be
+changed by modifying the options passed to `ObserveNSlabs` and `ObserveAtT0`.
+A successful observation will result in the creation of H5 files whose names
+can be specified in the YAML file under the options `VolumeFileName` and
+`ReductionFileName`. One volume data file will be produced from each Charm++
+node that is used to run the executable. Each volume data file will have its
+corresponding node number appended to its file name. Visualization of the
+volume data will be described in the next section.
+
+### 3D %Data Volume %Data In ParaView
+
+A SpECTRE executable with observers produces volume and/or reduced data h5
+files. An XDMF file must be created from the volume data in order to do
+visualization using ParaView. To this end we provide the python executable
+`GenerateXdmf.py` in the `tools` directory. The `--file-prefix` and `--output`
+arguments must be specified when the command is run; they correspond to the
```

Comment:
Split the description of the options `file-prefix` and `output` into separate sentences. (Put the part about leaving out the node number into the sentence for `file-prefix`